src/connectors/geo-search/connectGeoSearch.js

```diff
@@ -171,7 +171,10 @@ const connectGeoSearch = (renderFn, unmountFn) => {
 
     const toggleRefineOnMapMove = () =>
       widgetState.internalToggleRefineOnMapMove();
-    const createInternalToggleRefinementonMapMove = (render, args) => () => {
+    const createInternalToggleFacetRefinementOnMapMove = (
```

Comment:
To revert (see https://github.com/algolia/instantsearch.js/pull/3207#discussion_r226257220).